apps/transport/priv/gettext/en/LC_MESSAGES/explore.po

```diff
@@ -0,0 +1,20 @@
+## "msgid"s in this file come from POT (.pot) files.
+##
+## Do not add, change, or remove "msgid"s manually here as
+## they're tied to the ones in the corresponding POT file
+## (with the same domain).
+##
+## Use "mix gettext.extract --merge" or "mix gettext.merge"
+## to merge POT files into PO files.
+msgid ""
+msgstr ""
+"Language: en\n"
+"Plural-Forms: nplurals=2\n"
+
+#, elixir-autogen, elixir-format
+msgid "GTFS-RT vehicule positions"
+msgstr "Vehicle positions (GTFS-RT) - beta"
+
+#, elixir-autogen, elixir-format
+msgid "beta-explanation"
+msgstr "This maps aggregates all the GTFS-RT files referenced on this site, and display their vehicle positions in realtime."
```

Comment:
"this map" sans le "s". 
~J'aurais aussi plutôt dit "vehicles position".~ => google dit vehicle positions,  donc bon
Et realtime => plutôt "real-time" ou "real time" https://en.wikipedia.org/wiki/Real-time

Reply:
Traité dans f0bfe9986401396452b664f63fd3ef1b40242f49, merci !